agent/engine/host_resource_manager.go

```diff
@@ -139,7 +139,10 @@ func (h *HostResourceManager) consume(taskArn string, resources map[string]*ecs.
 		logger.Info("Resources successfully consumed, continue to task creation", logger.Fields{"taskArn": taskArn})
 		return true, nil
 	}
-	logger.Info("Resources not consumed, enough resources not available", logger.Fields{"taskArn": taskArn})
+	logger.Info("Resources not consumed, enough resources not available", logger.Fields{
+		"taskArn":  taskArn,
+		"resource": failedResourceKey,
```

Comment:
Just to note, there can be cases when more than one resources are not available. This log statement will print the first one `h.consumable` exits one. I think can be still okay to log it, although which resource gets logged is probably not deterministic.

Reply:
good callout!

Reply:
I incorporated this feedback, now we log all non-consumable resources. please take a look

Reply:
lgtm, thank you!